94.md

```diff
@@ -0,0 +1,49 @@
+NIP-94 - File Header
+======
+`draft` `optional` `author:frbitten` 
+
+The purpose of this NIP is to allow an organization and classification of shared files. So that relays can filter and organize in any way that is of interest.
+Also the goal is to create a base on the protocol for this bountsr "Filesharing App" (https://bountsr.org/p2p-filesharing/) to be implemented.
+
+Nostr event
+------------------
+This NIP specifies the use of the `1063` event type (parameterized replaceable event), having in `content` a description of the file content, and a list of tags described below:
+* `url` the url to download the file
```

Comment:
`r` is already used elsewhere as "reference" with url value.

The reason to make it searchable is that I imagined a client may want to have an user media panel that the user would fill by using a context menu pop-up over images on notes (like "add to memes collection" for instance).

For that, it would detect used image url on kind 1 note content and create a corresponding kind 1063 event. Making 1063 url tag searchable will allow the client to know if an event with that url from the same pubkey already exists.

It would be an alternative to searching by hash.

```suggestion
* `r` the url to download the file, without trailing slash
```